permissions/permissions-cg.https.html

```diff
@@ -7,20 +7,44 @@
 <script src="/resources/testdriver-vendor.js"></script>
 
 <script>
+async function garbageCollect() {
+  if (typeof TestUtils !== 'undefined' && TestUtils.gc) {
+    await TestUtils.gc();
+  } else if (self.gc) {
+    await self.gc();
+  } else if (self.GCController) {
+    // Present in some WebKit development environments
+    await GCController.collect();
+  } else {
+    var gcRec = function (n) {
+      if (n < 1)
+        return {};
+      var temp = {i: "ab" + i + (i / 100000)};
+      temp += "foo";
+      gcRec(n-1);
+    };
+    for (var i = 0; i < 1000; i++)
+      gcRec(10);
+  }
+}
+
```

Comment:
I don't think `TestUtils.gc` is a thing... at least, it's not in the tree that I can find. 

Maybe just: 
```suggestion
async function garbageCollect() {
  if (self.gc) return self.gc();
  // Present in some WebKit development environments
  if (self.GCController) return GCController.collect();

  for (var i = 0; i < 1000; i++) gcRec(10);

  function gcRec(n) {
    if (n < 1) return {};
    let temp = { i: "ab" + i + i / 100000 };
    temp += "foo";
    gcRec(n - 1);
  }
}

```

Reply:
https://testutils.spec.whatwg.org/#the-testutils-namespace

Note that I copied this function from another WPT test.

Reply:
Ok, added it back.